cli/tests/integration_tests.rs

```diff
@@ -2764,6 +2764,14 @@ mod util {
       return false;
     }
 
+    let pattern_first_line = pattern.split("\n").collect::<Vec<&str>>()[0];
+    // If the first line of the pattern is just a wildcard the newline character
+    // needs to be pre-pended so it can safely match anything or nothing and
+    // continue matching.
+    if pattern_first_line == wildcard {
```

Comment:
maybe use `pattern.lines().next() == Some(wildcard)`

Reply:
Yeah that looks a lot cleaner